src/Sulu/Bundle/ContactBundle/Resources/public/js/services/account-manager.js

```diff
@@ -250,8 +258,10 @@ define([
                 requests.push(removeAccountContact(accountId, id));
             }.bind(this));
 
-            $.when.apply(null, requests).then(function() {
+            $.when.apply(null, requests).done(function() {
```

Comment:
@wachterjohannes this is the proper way to wait for a deferred-resolve. (see: https://api.jquery.com/category/deferred-object/)